src/ui/public/pattern_checker/pattern_checker.html

```diff
@@ -1,2 +1,2 @@
 <span class="spinner"></span>
-Querying {{checker.pattern}}... {{checker.resultCount}} results
```

Comment:
Yeah I think adding the <pre> around it per the Invision mockup helped, but it could probably be improved further.